connector/src/main/java/org/geysermc/connector/network/cidr/CIDRMatcher.java

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Not sure if we need a whole package for this class. I would just throw this in the `network` or `utils` package but I'll get some second opinions.